### PR TITLE
Added Highlight completed Commissions

### PR DESCRIPTION
**Pull request by CuzImClicks**

### CODE REVIEW

File: src/main/java/me/Danker/commands/SetAngleCommand.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    public int getRequiredPermissionLevel() {
+        return 0;
+    }
+
+    @Override
+    public void processCommand(ICommandSender sender, String[] args) throws CommandException {
+        if (args.length < 2) return;
+
+        Minecraft.getMinecraft().thePlayer.rotationPitch = Float.parseFloat(args[1]);
+        Minecraft.getMinecraft().thePlayer.rotationYaw = Float.parseFloat(args[0]);
```
Comment (RabbitType99) [inline src/main/java/me/Danker/commands/SetAngleCommand.java:37]: I am sure that this counts as is an unfair-game-advantege and is bannable

File: src/main/java/me/Danker/DankersSkyblockMod.java [hunk 2]
```diff
@@ -393,7 +404,7 @@ public void onGuiMouseInputPre(GuiScreenEvent.MouseInputEvent.Pre event) {
 
     @SubscribeEvent
     public void onGuiRender(GuiScreenEvent.BackgroundDrawnEvent event) {
-        if (!Utils.inSkyblock) return;
+        //if (!Utils.inSkyblock) return;
```
Comment (bowser0000) [inline src/main/java/me/Danker/DankersSkyblockMod.java:407]: Uncomment this

File: src/main/java/me/Danker/commands/DungeonsCommand.java [hunk 3]
```diff
@@ -1,152 +0,0 @@
...
-import net.minecraft.command.ICommandSender;
-import net.minecraft.entity.player.EntityPlayer;
-import net.minecraft.event.HoverEvent;
-import net.minecraft.util.BlockPos;
-import net.minecraft.util.ChatComponentText;
-import net.minecraft.util.EnumChatFormatting;
-
-import java.util.List;
-
-public class DungeonsCommand extends CommandBase {
```
Comment (bowser0000) [inline src/main/java/me/Danker/commands/DungeonsCommand.java:19]: Why was this command deleted?

File: src/main/java/me/Danker/commands/SetAngleCommand.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+import me.Danker.DankersSkyblockMod;
+import net.minecraft.client.Minecraft;
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.util.ChatComponentText;
+
+
+public class SetAngleCommand extends CommandBase {
```
Comment (bowser0000) [inline src/main/java/me/Danker/commands/SetAngleCommand.java:11]: Headsnapping is detectable by watchdog and would probably result in a ban.

File: src/main/java/me/Danker/commands/warp/Crypt.java [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    public List<String> addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos) {
+        return null;
+    }
+
+    @Override
+    public void processCommand(ICommandSender sender, String[] args) throws CommandException {
+        if (!Utils.inSkyblock) return;
+        EntityPlayer player = ((EntityPlayer) sender);
+        // MULTI THREAD DRIFTING
+        new Thread(() -> {
```
Comment (bowser0000) [inline src/main/java/me/Danker/commands/warp/Crypt.java:43]: Don't know if sending a chat message creates enough lag to require making a new thread, same applies for every other warp command.

File: src/main/java/me/Danker/features/HighlightCommissions.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+        if(!ToggleCommand.highlightCommissions) return;
+        List<Slot> slots = event.slots;
+        if (!event.displayName.equals("Commissions")) return;
+
+        for (Slot slot : slots) {
+            if (slot != null && slot.getStack() == null) continue;
+            if (slot.getStack().getItem() instanceof ItemWritableBook) {
+                for (String line : Utils.getItemLore(slot.getStack())) {
+                    if (line.contains("COMPLETED")) {
+                        Utils.drawOnSlot(event.chestSize, slot.xDisplayPosition, slot.yDisplayPosition, 0x51FF51 + 0xD7000000);
```
Comment (bowser0000) [inline src/main/java/me/Danker/features/HighlightCommissions.java:28]: Colour should be added to config.

**Review by bowser0000 — CHANGES_REQUESTED** [review]

File: src/main/java/me/Danker/commands/SetAngleCommand.java (hunk 4 above)
Comment (CuzImClicks) [reply src/main/java/me/Danker/commands/SetAngleCommand.java:11]: Is there a way to ask if such a feature is actually bannable?

**Comment by CuzImClicks** [comment]
@bowser0000 i changed some stuff could you review again please?

**Pull request merged**